src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -38,7 +43,7 @@ public OpenCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> path
                 || !argMultimap.getPreamble().isEmpty()) {
             titles = args.trim();
```

Comment:
Do you want to rename `titles` as `title`, since it is actually just 1 title (which may consist of multiple String)?

Reply:
Sorry for not nitpicking this earlier, but can just use replace all feature hehe

Reply:
Oh i thought a path can have many titles.